Main.java

```diff
@@ -0,0 +1,722 @@
+package Main.java;

```

Comment:
you shouldn't have all the classes in Main. the criteria was to create packages with classes for entities, services, Main as a separate class that calls the services' methods. please fix this.